components/locale_core/src/extensions/transform/fields.rs

```diff
@@ -154,6 +158,7 @@ impl Fields {
     /// assert_eq!(old_value, Some(casefold));
     /// assert_eq!(loc, "en-t-hi-d0-lower".parse().unwrap());
     /// ```
+    #[cfg(feature = "alloc")]
```

Comment:
Nit: According to our style guide, please add Cargo feature warning comments on these methods... unless cargo-doc has fixed the long-standing bug?

Reply:
We do it for `compiled_data` which is a default feature, amongst others